```
arm-none-eabi/bin/ld: /test/build/test/release/test.elf section `.data' will not fit in region `FLASH'
arm-none-eabi/bin/ld: region `FLASH' overflowed by 76 bytes
collect2: error: ld returned 1 exit status
```

@rleh I guess there's a small regression in the new GCC version that makes the flash overflow on the F103 unittests. I've fixed this in #343.

--- a/src/modm/platform/timer/stm32/general_purpose.hpp.in
+++ b/src/modm/platform/timer/stm32/general_purpose.hpp.in
@@ -357,11 +357,13 @@ public:
 	/// @param channel may be [1..4]
 	%% elif (id in [8, 9, 12])
 	/// @param channel may be [1..2]
+	%% else
+	/// @param channel is not used
 	%% endif
 	/// @return `false` if configured as *output*; `true` if configured as *input*
 	%% if (id in [10, 11, 13, 14])
 	static inline bool
-	isChannelConfiguredAsInput()
+	isChannelConfiguredAsInput([[maybe_unused]] uint32_t channel = 1)
 	{
 		return TIM{{ id }}->CCMR1 & TIM_CCMR1_CC1S;
 	}

